Makefile

```diff
@@ -2,8 +2,18 @@ RELEASE_REPO ?= public.ecr.aws/karpenter
 RELEASE_VERSION ?= $(shell git describe --tags --always)
 RELEASE_PLATFORM ?= --platform=linux/amd64,linux/arm64
 
+# https://reproducible-builds.org/docs/source-date-epoch/
+DATE_FMT = %Y-%m-%dT%H:%M:%SZ
```

Comment:
```suggestion
DATE_FMT = +%Y-%m-%dT%H:%M:%SZ
```

Reply:
great catch, I used vim to do updates, sometimes I catch those things 🙋🏻‍♂️ Thanks for having my back 🤝